### PR TITLE
Added internet explorer implementation and NTLM firefox profile option

### PR DESCRIPTION
**Pull request by roydekleijn**
please review

is there a code style import file specific for Apache Jmeter?

### CODE REVIEW

**Comment by roydekleijn** [comment]
@cplim I changes the files you suggested

There is also an issue with the maven-shade-plugin-2.0 plugin. Can it be updated to 2.1? (issue is solved in that version)

File: webdriver/pom.xml [hunk 1]
```diff
@@ -263,8 +268,8 @@
                             <overWriteReleases>false</overWriteReleases>
                             <overWriteSnapshots>false</overWriteSnapshots>
                             <overWriteIfNewer>true</overWriteIfNewer>
-                            <includeArtifactIds>selenium-support,selenium-api,selenium-remote-driver,selenium-firefox-driver,selenium-chrome-driver,selenium-htmlunit-driver,
-                            	guava,json,commons-exec,jsyntaxpane,htmlunit,htmlunit-core-js,nekohtml,cssparser,jetty-websocket,xercesImpl,xml-apis,sac,httpclient,httpmime,httpcore,gson
```
Comment (cplim) [inline webdriver/pom.xml]: The removal of the **gson** library breaks webdriver lib. @roydekleijn, can you update your pull request to include this?

Comment (roydekleijn) [inline webdriver/pom.xml]: sorry, wasn't aware of the removal..

**Comment by cplim** [comment]
Thanks for the feature. Apologies once again for the late reply.

**Comment by undera** [comment]
Guys, didn't you forget the changelog records? It is required...

**Pull request merged**